velox/connectors/hive/HiveConfig.h

```diff
@@ -91,10 +96,16 @@ class HiveConfig {
   static constexpr const char* kGCSCredentials = "hive.gcs.credentials";
 
   /// Maps table field names to file field names using names, not indices.
+  // TODO: remove hive_orc_use_column_names since it doesn't exist in presto,
+  // right now this is only used for testing.
   static constexpr const char* kOrcUseColumnNames = "hive.orc.use-column-names";
+  static constexpr const char* kOrcUseColumnNamesSession =
+      "hive_orc_use_column_names";
 
   /// Reads the source file column name as lower case.
   static constexpr const char* kFileColumnNamesReadAsLowerCase =
+      "file-column-names-read-as-lower-case";
+  static constexpr const char* kFileColumnNamesReadAsLowerCaseSession =
       "file_column_names_read_as_lower_case";
```

Comment:
Hi @kewang1024 

The change broke some tests in our project that relies on Velox and then took us some time to debug. Would you like to share the reason why having to rename config key `kFileColumnNamesReadAsLowerCase` ?

Reply:
Hi, this is an effort to separate config from session property. 
And fix the config name convention to be "-" and session property name convention "_", otherwise people are using them sometime "randomly" and it has caused a lot of bugs for us. Also you can refer to the issue: https://github.com/facebookincubator/velox/issues/7659